google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -255,13 +255,24 @@ public boolean equals(Object o) {
 
     abstract ImmutableList<FieldReference> getFieldProjections();
 
+    // Whether to select all documents under `parentPath`. By default, only
+    // collections that match `collectionId` are selected.
+    abstract boolean getKindless();
```

Comment:
https://www.youtube.com/watch?v=AylaVaBlgCo

Reply:
This should be `isKindless`: https://stackoverflow.com/questions/5322648/for-a-boolean-field-what-is-the-naming-convention-for-its-getter-setter